megamek/src/megamek/common/BipedMech.java

```diff
@@ -172,6 +172,10 @@ public int getWalkMP(boolean gravity, boolean ignoreheat, boolean ignoremodulara
             if (weatherMod != 0) {
                 wmp = Math.max(wmp + weatherMod, 0);
             }
+            if(getCrew().getOptions().stringOption(OptionsConstants.MISC_ENV_SPECIALIST).equals(Crew.ENVSPC_WIND)
+                    && game.getPlanetaryConditions().getWeather() == PlanetaryConditions.WI_TORNADO_F13) {
```

Comment:
```suggestion
            if (getCrew().getOptions().stringOption(OptionsConstants.MISC_ENV_SPECIALIST).equals(Crew.ENVSPC_WIND)
                    && (game.getPlanetaryConditions().getWeather() == PlanetaryConditions.WI_TORNADO_F13)) {
```

Same, what about stronger wind?

Reply:
F4 for mech is walk only